backend/src/packages/chaiNNer_standard/image/video_frames/save_video.py

```diff
@@ -98,6 +98,62 @@ class AudioSettings(Enum):
     TRANSCODE = "transcode"
 
 
+class Simplicity(Enum):
+    SIMPLE = 0
+    ADVANCED = 1
+
+
+class SimpleVideoFormat(Enum):
+    MP4_H264 = "mp4_h264"
+    MP4_H265 = "mp4_h265"
+    WEBM = "webm"
+    GIF = "gif"
+
+
+def get_simple_format(
+    simple_video_format: SimpleVideoFormat, quality: int
+) -> tuple[VideoFormat, VideoEncoder, VideoPreset, int]:
+    container = None
+    encoder = None
+    video_preset = None
+    crf = None
+
+    container = {
+        SimpleVideoFormat.MP4_H264: VideoFormat.MP4,
+        SimpleVideoFormat.MP4_H265: VideoFormat.MP4,
+        SimpleVideoFormat.WEBM: VideoFormat.WEBM,
+        SimpleVideoFormat.GIF: VideoFormat.GIF,
+    }[simple_video_format]
+
+    if simple_video_format != SimpleVideoFormat.GIF:
+        encoder = {
+            SimpleVideoFormat.MP4_H264: VideoEncoder.H264,
+            SimpleVideoFormat.MP4_H265: VideoEncoder.H265,
+            SimpleVideoFormat.WEBM: VideoEncoder.VP9,
+        }[simple_video_format]
+    else:
+        # Encoder will be ignored, it just needs to be a value in the case of GIF
+        encoder = VideoEncoder.H264
```

Comment:
This feels like the logic is backwards. I think of GIF as the special case, but this looks like GIF is related to the general case instead. I would turn that around like this:

```suggestion
    if simple_video_format == SimpleVideoFormat.GIF:
        # Encoder will be ignored, it just needs to be a value in the case of GIF
        encoder = VideoEncoder.H264
    else:
        encoder = {
            SimpleVideoFormat.MP4_H264: VideoEncoder.H264,
            SimpleVideoFormat.MP4_H265: VideoEncoder.H265,
            SimpleVideoFormat.WEBM: VideoEncoder.VP9,
        }[simple_video_format]
```

Reply:
ended up just adding a case to the dict for gif